tests/atf_python/sys/net/vnet.py

```diff
@@ -180,6 +180,7 @@ def create_iface(self, alias_name: str, iface_name: str) -> List[VnetInterface]:
 
     @staticmethod
     def is_autodeleted(iface_name: str) -> bool:
+        return "lo0" not in iface_name
```

Comment:
Doesn't this unconditionally return, i.e., the subsequent lines are never executed?

Reply:
yeah, this would need an `if` to be actually conditional.

Reply:
Yes. The overall way it works is that for each interface, it checks if `is_autodeleted`, so I wan it to return `False` if the interface is `lo0`, it will call this method once for each interface.
Previously any loopback interface was added to the list of interfaces to be automatically deleted.
This change will produces the `lo0` to be skipped, for example:
```
...
sbin/ping/test_ping.py::TestPing::test_pinger[_0_0_special_wrong] ==== vnet cleanup ====
# topology_id: 'TestPing:test_pinger[_0_0_special_wrong]'
Skipping interface pytest:TestPing:test_pinger__0_0_special_wrong:lo0
run: '/usr/sbin/jexec pytest:TestPing:test_pinger__0_0_special_wrong ifconfig epair1a destroy'
run: '/usr/sbin/jexec pytest:TestPing:test_pinger__0_0_special_wrong ifconfig tun0 destroy'
run: '/usr/sbin/jail -r pytest:TestPing:test_pinger__0_0_special_wrong'
PASSED

```

Reply:
I'll re-submit.